aws/local.yml

```diff
@@ -8,6 +8,7 @@
     - "environments/{{ stack_name }}.yml"
   roles:
     - common
+    - web
```

Comment:
By explicitly listing this role early, we make nginx come up slightly earlier in the build process, which helps the "waiting" checks above proceed in a more timely fashion.